dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -51,7 +51,10 @@ class SnowflakeMicrobachelorsITK {
                 }
             }
             triggers {
-                cron('H 6 * * 1-5')
+                buildResult('H 6 * * 1-5') {
+                    combinedJobs()
+                    triggerInfo('warehouse-transforms-microbachelors', BuildResult.SUCCESS)
```

Comment:
Does this groovy script know about warehouse-transforms-microbachelors? That’s the job name in Jenkins but that name is generated from some parameterization, I think.

Do I have to indicate BuildResult.SUCCESS or is that the default?